ha/fault_tolerance/event_manager.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
```

Comment:
This directory structure is mismatch

 ha/fault_tolerance/event_manager.py
 ha/core/event_manager/error.py